src/core_plugins/kibana/ui_setting_defaults.js

```diff
@@ -122,6 +122,10 @@ export function getUiSettingDefaults() {
       value: 100,
       description: 'Never show more than this many bars in date histograms, scale values if needed',
     },
+    'visualize:labMode': {
```

Comment:
Why do we call it `labMode`? Also I think the description is pretty much saying nothing. "Enable lab mode in visualize" leaves the user with the question: what is lab mode? What will enable/disable this option actually cause?

I would rather rename the setting `visualize:enableExperimentalVis` and change the description to something like: "Enable experimental (not yet stable) visualizations in Kibana".